### PR TITLE
fix: rerun metric

### PR DESCRIPTION
**Pull request by wildeslam**

### CODE REVIEW

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java [hunk 1]
```diff
@@ -163,19 +165,41 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
...
     ComparisonInterfaceConfig operationConfig = compareConfigService.loadInterfaceConfig(
         caseItem.getParent());
-    List<String> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
+    List<CategoryDetail> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
 
     List<ReplayCompareResult> replayCompareResults = new ArrayList<>();
     for (CategoryComparisonHolder bindHolder : waitCompareMap) {
       if (operationConfig.checkIgnoreMockMessageType(bindHolder.getCategoryName(),
-          ignoreCategoryList)) {
+          caseItem.getParent().getOperationName(), ignoreCategoryList)) {
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java:173]: In scenarios where the main interface is not allowed to be ignored, there is no need to pass in operationName.

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java [hunk 2]
```diff
@@ -163,19 +165,41 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
...
         continue;
       }
+      filterIgnoredCompareItem(bindHolder.getReplayResult(), operationConfig);
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java:176]: suggestion: put the filtering of record and replay data in one method

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java [hunk 3]
```diff
@@ -163,19 +165,41 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
...
+      filterIgnoredCompareItem(bindHolder.getRecord(), operationConfig);
       replayCompareResults.addAll(compareReplayResult(bindHolder, caseItem, operationConfig));
     }
     return replayCompareResults;
   }
 
+  private void filterIgnoredCompareItem(List<CompareItem> replayResults,
+      ComparisonInterfaceConfig operationConfig) {
+    List<CategoryDetail> ignoreCategoryTypes = operationConfig.getIgnoreCategoryTypes();
```
Comment (coryhh) [inline arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java:185]: The upper method already has "getIgnoreCategoryTypes", and it feel like you can pass in ignoreCategoryTypes directly.

Comment (wildeslam) [reply arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java:185]: fixed

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java (hunk 2 above)
Comment (wildeslam) [reply arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java:176]: accepted

File: arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java (hunk 1 above)
Comment (wildeslam) [reply arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java]: fixed

**Review by coryhh — APPROVED** [review]

**Review by QizhengMo — APPROVED** [review]

**Pull request merged**